requirements/dev.in

```diff
@@ -9,6 +9,7 @@ responses==0.10.9
 django-extensions==2.2.8
 pytest-selenium==1.17.0
 PyPOM==2.2.0
+pre-commit==2.2.0
```

Comment:
@imskr imho, pre-commit is an external dependency. We do not intend it to keep it installed along with dependency packages, as it an optional method

Reply:
Since it is now part of Travis it makes now a bit more of sense to add it here.

Ignore the request to re-review. I did that by mistake.